.gitattributes

```diff
@@ -1,5 +1,6 @@
 * text=auto eol=lf
-test/corpus/c/crlf.txt text eol=crlf
+
+test/corpus/c/crlf.txt eol=crlf
```

Comment:
Why modify line endings in gitattributes? This will change the eol of the file unless it already has crlfr line endings.

Reply:
It **must** have crlf line endings.